text/0000-export-to-oci.md

```diff
@@ -0,0 +1,213 @@
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+When the `Exporter` phase is invoked passing the `-daemon` flag  besides writing into the Daemon also save the image to disk in [OCI Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format updating the [report.toml](https://github.com/buildpacks/spec/blob/main/platform.md#reporttoml-toml) file with all the metadata require to validate the consistency of the image when it is published to a Registry.
+
+# Definitions
+[definitions]: #definitions
+
+- A [Platform](https://buildpacks.io/docs/concepts/components/platform/) uses a lifecycle, Buildpacks (packaged in a builder), and application source code to produce an OCI image.
+- A [Lifecycle](https://buildpacks.io/docs/concepts/components/lifecycle/) orchestrates Buildpacks execution, then assembles the resulting artifacts into a final app image.
+- A Daemon is a service, popularized by Docker, for downloading container images, and executing and managing containers from those images.
+- A Registry is a long-running service used for storing and retrieving container images.
+- A digest reference refers to a [content addressable](https://en.wikipedia.org/wiki/Content-addressable_storage) identifier of form <registry>/<repo>@<digest> which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+- A Image Manifest provides a configuration and set of layers for a single container image for a specific architecture and operating system.
+- An [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) is the directory structure for OCI content-addressable blobs and location-addressable references.
+
+# Motivation
+[motivation]: #motivation
+
+Implementing this new feature will help us to solve the problem of loosing information when the image is saved into the Daemon keeping the image on disk along with the metadata it can be used as input for other tools to offer more capabilities to the end users.
+
+This feature will help to unblock uses cases like
+- OCI annotations. See [RFC](https://github.com/buildpacks/rfcs/pull/196)
+- Cosign integration. See [RFC](https://github.com/buildpacks/rfcs/pull/195)
+
+# What it is
+[what-it-is]: #what-it-is
+
+Currently the *Exporter*  writes either in an OCI image registry or a docker daemon, the idea is to add the capability to write into disk in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format only when the `-daemon` flag is used as argument AND the feature is enable using a new flag `-layout` or the default environment variable `CNB_LAYOUT_DIR`.
```

Comment:
We'll need to get the run image digest from the daemon, which may or may not be available: https://github.com/buildpacks/imgutil/issues/32

Should we assume that the digest is always available, and fail otherwise?